conf/checksums

```diff
@@ -1,5 +1,6 @@
 accumulo:2.1.0:9bbc4defc114013f145e9e9fdd08683c842a29faaa01ea6e80555549a6aecef86ee6657cce23183b411a64c31123db682a6944e2825eef83c76fb5a91620235f
 accumulo:2.0.1:b443839443a9f5098b55bc5c54be10c11fedbaea554ee6cd42eaa9311068c70bd611d7fc67698c91ec73da0e85b9907aa72b98d5eb4d49ea3a5d51b0c6c5785f
+accumulo:1.10.3:36e6795ad3720ba72fc9f4ddabf45f6d67cdc77658a181733fa2c47bfd3799f123b8840a79b538b8504a7b4bdc97fd0b52efab790395a5e022a897bc18405d0c
 accumulo:1.10.2:9b6e9286133120588f4c682e3fadee33704cf5983ef7e3c0be84c1678bd9bcf283a2c6efa6d4074c73f30032f5c0aa1073c15ec39f90d215505e028a1cf0a739
```

Comment:
Should only include the latest bugfix release for a given series. Can drop 1.10.2

```suggestion
```